core/src/sigverify.rs

```diff
@@ -342,6 +387,60 @@ mod tests {
         assert_eq!(sig_len, 1);
     }
 
+    #[test]
+    fn test_untrustworthy_sigs() {
+        let required_num_sigs = 14 as u32;
```

Comment:
don't need this cast and the one below, compiler will figure out u8 type.

Reply:
Nice! a525f96880a02ca1f937a5fbba73c673e0f22c26